script.js

```diff
@@ -1,4 +1,145 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+
+
+// There are 2 game modes
+var gameModeDiceRoll = 'gameModeDiceRoll';
+var gameModeDiceOrder = 'gameModeDiceOrder';
+
+//for each roll of two independent dice
+var rollOfDice = [];
+
+//To keep track of each player playerCombineNumber
+var playerCombineNumber;
+var playerCombineNumber1 = [];
+var playerCombineNumber2 = [];
+
+//Start game with Player 1
+var currPlayer = 1;
+
+//variable for winner
+var winner = "";
+
+//Start game with gameModeDiceRoll
+var gameMode = gameModeDiceRoll;
+
+var main = function (input) { 
+  if (input.toLowerCase() == "r"){
+    var finalScore1 = player1Result();
+    var finalScore2 = player2Result();
+
+    if(finalScore1 > finalScore2){
+      winner = 1;
+    }
+    else if (finalScore2 > finalScore1){
+      winner = 2;
+    }
+    return `Winner is Player ${winner} <br> Player 1 score is ${finalScore1} <br> Player 2 score is ${finalScore2}`;
+  };
+
+  if (gameMode == gameModeDiceRoll){
+      rollOfDice = getDiceRolls();
+      console.log(rollOfDice);
+      
+      //switch to gameModeDiceOrder
+      gameMode = gameModeDiceOrder;
+      
+      var myOutputValue = 
+      `Welcome Player ${currPlayer} <br>
+      You rolled: <br>
+      Dice 1 -- ${rollOfDice[0]}  <br>
+      Dice 2 -- ${rollOfDice[1]}  <br>
+      Choose the order of the first dice <br>
+      Dice 1 -- Enter 1 <br>
+      Dice 2 -- Enter 2`;
+      return myOutputValue;
+  }
+  //for gameModeDiceOrder, assign user input to dice order
+  if (gameMode == gameModeDiceOrder)
```

Comment:
Missed an opening {  here and a closing } between lines 102 and 103. You got a little lucky that the code still ran as you missed both 😬